tests/steps/state_get_balance_test.go

```diff
@@ -65,6 +65,8 @@ func InitializeStateGetBalance(ctx *godog.ScenarioContext) {
 	ctx.Step(`^the state_get_balance_result contains the purse amount$`, func() error {
 		var expectedBalance big.Int
 		var purseUref string
+		val := big.Int{}
```

Comment:
Does val need to be declared here. Could be within the braces it's being used in